robots/cmd/per-test-execution/main.go

```diff
@@ -109,7 +110,23 @@ func (o options) loadDefaults() error {
 		if !k8sStableReleaseVersionRegex.MatchString(k8sStableReleaseVersion) {
 			return fmt.Errorf("Kubernetes stable version %q doesn't match regex", k8sStableReleaseVersion)
 		}
-		opts.K8sVersion = k8sStableReleaseVersionRegex.FindAllStringSubmatch(k8sStableReleaseVersion, -1)[0][1]
+		// Align the report to current sig-compute-migrations
+		retryCounter := 3
```

Comment:
This is not really retry as far as I understand, is it?

Reply:
Good catch - I missed handling the case where the `retryCounter == 0` will make the for loop break. I'll rework the code to explicitly return an error if above condition is true. Also I'll rename the variable to make it's name reflect what it does.

Thanks!

Reply:
@xpivarc updated - PTAL, thank you! :pray: